src/cutapplier.cc

```diff
@@ -22,9 +22,10 @@ using std::endl;
 
 int main(int argc, char *argv[]) {
 	if(argc != 5 && argc != 6){
-		cout << "cutapplier:   	Applies a cut to an input ntuple"<< endl;
-		cout << "author:        Conor Fitzpatrick, 2008"<< endl;
-		cout << "Syntax: " << argv[0] << " <input.root> <path/to/ntuple> <cut string> <accepted_output.root> [<input_friend.root>]"<< endl;
+		cout << "cutapplier:          Applies a cut to an input ntuple"<< endl;
+		cout << "author:              Conor Fitzpatrick, 2008"<< endl;
+		cout << "Syntax:              " << argv[0] << " <input.root> [<input_friend.root>] <path/to/ntuple> <cut string> <accepted_output.root>"<< endl;
+		cout << "<input_friend.root>: Optional, friend added to <input.root>. Not merged into <accepted_output.root>" << endl;
```

Comment:
Hi @jacopocerasoli Thanks for making these changes. Just to check, the purpose of the friend tree is to provide additional parameters to be cut on that are not present in the original tree, and this friend tree is always expected to have the same path as the input tree? Is this by construction? Suggest this is noted in the arguments.

Reply:
Hi @cofitzpa Yes that's correct, you can use the tree to cut on variables which are not present in the original tree. I also decided to leave the structure of the original tree unchanged i.e. the variables in the friend are not merged into the original tree. If you want the code to be more flexible I could also add a different path for the tree in the friend, the syntax becoming something like `cutapplier input.root path/to/tree input_friend.root path/to/other/tree cut_string accepted_output.root`.

Reply:
I don't think there's a need at this stage and it's easy enough to do so in an additional MR later, so I'm happy to proceed as-is. I think making it clear in the output instructions is important though.